src/model.ts

```diff
@@ -381,13 +381,15 @@ export function build<TSchema extends BaseSchema, TOptions extends SchemaOptions
       if (operations.length === 0) {
         return;
       }
-      const finalOperations = operations.map((op) => {
+
+      const finalOperations = [];
```

Comment:
can we use `Promise.all` here?

Reply:
It would be possible if we refactor this part a bit, but I'm not sure if it's worthy.

The key point here is that we need to compute a different default value for each operation. I think you are asking because you want to keep .map to iterate, but that method doesn't await for async functions. So converting to Promise.all means one iteration to call getDefaultValues and wait for the values. Then another iteration for merging the operation with the default values. 

Also take into account that the default values are not computed in all bulk operations.

Reply:
I think what @ejmartin504 was referring to is the following:

```ts
const finalOperations = await Promise.all(operations.map(async (op) => {
	// code from the inner `for` loop
}));
```

While it's true that we may introduce new async cycles for operations that are not using the defaults, maybe we gain the speed back by running these operations in parallel.

I would go a step further and ask that we use something similar to `p-map` (but not `p-map` exactly due to the whole ESM blocker on Sindre's packages), which may allow us to control the concurrency of these parallel operations.

Reply:
Ah, I see what he meant! Yes, it's a good option too.